TypeCobol.Test/Parser/Programs/TypeCobol/FunctionsDeclaration.rdz.tcbl

```diff
@@ -122,9 +119,6 @@
       * ERROR only level-88 parameter items shall have an explicit level number
        DECLARE FUNCTION FunConditions PRIVATE
              INPUT 88 valid-gender VALUE 'F' 'M'
-                      gender PIC X(01)
-                   88  female VALUE 'F'
-                   01  male   VALUE 'M'
```

Comment:
Same here, keep this invalid syntax test. By removing it, you'll have an unintended conflict with the previous FunConditions declaration. The idea is to update the error messages but not necessarily change the test data.

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/pull/1669/commits/3c0f9d082a18ff1b641cb676ce9152a389f4f984